index.php

```diff
@@ -0,0 +1,24 @@
+<?php
+$db = new PDO('mysql:host=db;dbname=Collection','root','password');
+
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$collection = $db->query('SELECT `latin_name`,`common_name`,`height(cm)`,`cap_width(cm)`,`deadly` FROM `Collection`');
```

Comment:
Is your table called 'Collection' or is that the database name? The error I get when I run the code is: Fatal error: Uncaught Error: Call to a member function fetchAll() on boolean in /var/www/html/Collection/index.php:8 Stack trace: #0 {main} thrown in /var/www/html/Collection/index.php on line 8

I had this issue...